src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerTest.java

```diff
@@ -58,25 +66,26 @@
   private DatasetReportService datasetReportService;
 
   @Test
-  void processDataset_expectSuccess() throws Exception {
+  void processDatasetFromURL_expectSuccess() throws Exception {
```

Comment:
This is still a multipart file test like it was before.
A test with a url instead of a multipart file is missing.